operator/missing-role.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Are we waiting on the operator release for this? We could also change kuttl to use kustomize though we might be losing a lot of coverage if we do.

Reply:
I would not use kustomize and wait for operator release.